level/level.c

```diff
@@ -89,8 +90,8 @@ static void minotaur_fire(void *context, void* vmob) {
 
 static void umber_hulk_fire(void *context, void* vmob) {
     mobile *mob = (mobile*)vmob;
-    //TODO wait... now we're doing rands like this?!
-    if (rand()/(float)RAND_MAX > 0.8) { //TODO magic number
+    //TODO did I keep the logic the same?
```

Comment:
You flipped it but your version makes more sense. Should be `prob(0.2)` to get the same behavior.